### ISSUE TITLE
Should I be able to create a new issue from the closed tab?

### ISSUE THREAD

**Issue by DanAbney** [BaoTon/testrepo#9]
technically it shouldn't matter since a new issue will be open, but because we are filtering and currently in the "Closed" tab, the new issue will not show up after it is submitted. Users could get confused...1. think their issue didn't create due to their own error and make another one. 2. think the system is buggy.

Do we automatically take them to "Open" view if they submit a new issue from closed?
or
Do we disable/hide 'Submit new' UI?
Or
Leave it as is?
      <hr><em> ⚠ Do not edit this section. It is required for imodeljs.github.io ➟ GitHub issue linking</em><br><br><ul><li>LINK: utterances-link-to-docs</li><li>TAG: imodeljs-docs-question</li> </ul>

**Comment by BaoTon** [comment]
The new issue shows in the "Open" Tab, I can remove the "Create Issue" tab in the closed section preferred.

You should be able to comment on a closed issue for further inquiry I believe.

**Comment by BaoTon** [comment]
You can test the "close" "reopen" feature too on issues you've made. It should move them between tabs.

**Comment by DanAbney** [comment]
Yeah, I think we should allow users to comment on closed issues. I will assume we are handling that well enough on our end (IE: is it very clear that they that the issue was previously closed in our view? AND closed issues do not fall off of our view somehow).
